sbt-cross-test/src/sbt-test/new-api/aggregates/foo/js/src/main/scala/Foo.scala

```diff
@@ -0,0 +1,5 @@
+object Foo {
+  def main(args: Array[String]): Unit = {
+    println(Bar.a)
+  }
+}
```

Comment:
created a super script for this:
`find -type f -name "*.scala" -print0 | xargs -0 -L1 bash -c 'test "$(tail -c 1 "$0")" && echo "$0"'`